renovate.json

```diff
@@ -2,5 +2,9 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "gomod": {
     "schedule": ["at any time"]
+  },
+  "prTitles": {
```

Comment:
I only see "prTitle" in the Renovate docs, not "prTitles".  "prTitle" looks like it would do what you want.